### PR TITLE
chronos now deploys correctly with dcos cli

### PR DESCRIPTION
**Pull request by swartzrock**
Includes healthchecks.

https://mesosphere.atlassian.net/browse/CHRONOS-43

### CODE REVIEW

**Comment by ssk2** [comment]
:+1:

**Comment by elingg** [comment]
Nice! I'm glad there are so many configurable options available

File: repo/packages/C/chronos/0/config.json [hunk 1]
```diff
@@ -1 +1,200 @@
-{}
+{
+  "type": "object",
+  "properties": {
+    "assets_path": {
```
Comment (ConnorDoyle) [inline repo/packages/C/chronos/0/config.json]: _All_ of these property names should be namespaced to the package.  E.g. `chronos/assets_path`.  See the Marathon example and try to conform to the precedent set there.

**Comment by swartzrock** [comment]
Thanks Connor, fixed them.

**Comment by ConnorDoyle** [comment]
This is great, thanks @swartzrock!

**Comment by jsancio** [comment]
@swartzrock @ConnorDoyle  It looks like this PR doesn't update the index.json is this expected or did we forget a step?

**Comment by ConnorDoyle** [comment]
I synced it after the fact.  We should add a check to the pre-commit hook to make sure there are no unstaged changes.

**Comment by elingg** [comment]
Minor nit, checkpointing is true by default.

**Pull request merged**